### PR TITLE
V0.2 - Add SSL wirdcard support

### PR DESCRIPTION
**Pull request by vdo**

### CODE REVIEW

**Comment by dapplion** [comment]
@vdo Can you split the PR into one that takes care of formatting and the other that actually changes the logic?

I'll suggest adding a this file in root https://github.com/dappnode/DNP_DAPPMANAGER/blob/master/build/src/.prettierrc
and then running this command on the whole project
https://github.com/dappnode/DNP_DAPPMANAGER/blob/aaa215081ddfe21ad83352b9dd766424c77b20d8/build/src/package.json#L20

**Comment by vdo** [comment]
> @vdo Can you split the PR into one that takes care of formatting and the other that actually changes the logic?
> 
> I'll suggest adding a this file in root https://github.com/dappnode/DNP_DAPPMANAGER/blob/master/build/src/.prettierrc
> and then running this command on the whole project
> https://github.com/dappnode/DNP_DAPPMANAGER/blob/aaa215081ddfe21ad83352b9dd766424c77b20d8/build/src/package.json#L20

The logic change is a minor one:
https://github.com/dappnode/dyndns-server/pull/3/files#diff-0a68b36677094791e9f2e58e36e4b778L8

I agree the lint makes a pain to see the changes, and I will take it into account but reverting this now is not worth the time for one line

File: api/src/util/nsupdate.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-    const cmdTimeout = 10 * 1000;
-    const nsupdateCmd = 'nsupdate -v'
-    var updateCmd = `server ${server}\ndebug yes\nzone ${zone}\nupdate delete ${address}.${zone} A\nupdate add ${address}.${zone} ${ttl} A ${ip}\nshow\nsend`
-    const res = await shell.echo(updateCmd).exec(nsupdateCmd,{timeout: cmdTimeout});
+async function nsupdate(server, zone, address, ttl, ip) {
+  //var dnsType = 'A'
+  shell.config.silent = true;
+  const cmdTimeout = 10 * 1000;
+  const nsupdateCmd = "nsupdate -v";
+  var updateCmd = `server ${server}\ndebug yes\nzone ${zone}\nupdate delete ${address}.${zone} A\nupdate add ${address}.${zone} ${ttl} A ${ip}\nupdate add \*.${address}.${zone} ${ttl} A ${ip}\nshow\nsend`;
```
Comment (eduadiez) [inline api/src/util/nsupdate.js]: should we add `delete \*.${address}.${zone}` also?

**Review by eduadiez — APPROVED** [review]
LGTM, although I guess we should also add '\*.' to delete

**Pull request merged**